reports/builder/2017-06-05.md

```diff
@@ -0,0 +1,62 @@
+# Development Report for June 5, 2017
+
+### New feature: Long running session
+
+Similarly to last week, the PR for [adding long-running session between daemon and cli](https://github.com/moby/moby/pull/32677) is waiting for reviews.  It is blocking many new features like the token signing, not pulling unnecessary context files, exposing sources outside working directory, etc. Maintainers are encouraged to give this one a review so it can be included in `v17.07` release.
+
+
+### Quality: Dependency interface switch
+
+Work continues on making the builder dependency interface more stable.
+
+PRs currently in review as part of this effort:
+
+- [Move file copying from the daemon to the builder](https://github.com/moby/moby/pull/33454)
+
+This PR is the core of the update that removes the need to track active containers and instead of lets builder hold references to layers while it's running.
+
+Related to this, @simonferquel opened a [WIP PR](https://github.com/moby/moby/pull/33492) that introduces typed Dockerfile parsing. This enables making [decisions about dependencies](https://github.com/moby/moby/issues/32550#issuecomment-297867334) between build stages and reusing Dockerfile parsing as a buildkit frontend.
+
+### Buildkit
+
+Some initial proof of concept code for [buildkit](https://github.com/moby/moby/issues/32925) has been pushed to https://github.com/tonistiigi/buildkit_poc . It's in a very early exploratory stage. Current codebase includes libraries for getting concurrency safe references to containerd snapshots using a centralized cache management instance. There is a sample source implementation for pulling images to these snapshots and executing jobs with runc on top of them. There is also some utility code for concurrent execution and progress stream handling. More info should follow in next weeks, including hopefully opening up an official repo. If you have questions or want to help, stop by the issues section of that repo or the proposal in moby/moby.
+
+### `v17.06` release
```

Comment:
This is not related to moby? Or should be "discovered downstream in Docker 17.06"